packages/persist/lib/server.ts

```diff
@@ -20,7 +21,7 @@ if (process.env['ENABLE_REQUEST_LOG'] !== 'false') {
     server.use(requestLoggerMiddleware({ logger }));
 }
 
-server.use('/environment/:environmentId/*', authMiddleware);
+server.use('/environment/:environmentId/*', authMiddleware, express.json());
```

Comment:
Not sure about the default, I think it might be 1mb, you could put a lower limit that fits those endpoints

Reply:
default is 100kb http://expressjs.com/uk/api.html#express.json
I think this value is fine